lib/opentelemetry/trace/exporter/circuit.lua

```diff
@@ -0,0 +1,77 @@
+local util = require("opentelemetry.util")
+local otel_global =  require("opentelemetry.global")
+
+local _M = {
+    OPEN = "open",
+    CLOSED = "closed",
+    HALF_OPEN = "half-open"
+}
+
+local mt = {
+    __index = _M
+}
+
+
+function _M.new(options)
+    options = options or {}
+    local self = {
+        circuit_reset_timeout_ms = options["circuit_reset_timeout_ms"] or 5000,
```

Comment:
simplify `circuit_reset_timeout_ms` to `reset_timeout_ms`

Reply:
a0872d549939f09cd84b2bc26e366c096495735e